src/test/unit/string.test.js

```diff
@@ -56,6 +60,12 @@ describe('utils/string', function() {
       expect(removeURLs(string)).toEqual('http://<URL>');
       string = 'https://foo_bar.baz.com/';
       expect(removeURLs(string)).toEqual('https://<URL>');
+      string = 'file://localhost/c$/WINDOWS/clock.avi';
+      expect(removeURLs(string)).toEqual('file://<URL>');
+      string = 'file:///c:/WINDOWS/clock.avi';
+      expect(removeURLs(string)).toEqual('file:///<URL>');
```

Comment:
ah I've seen this pattern: I think the last `/` is part of the URL and should be removed. I think we should always have `file://<URL>`. Do you think it's doable?